core/src/test/java/org/infinispan/statetransfer/RemoteGetDuringStateTransferTest.java

```diff
@@ -120,11 +177,16 @@ public void testScenario_010() throws Exception {
       rpcManager0.blockBefore(ClusteredGetCommand.class);
       topologyManager0.startBlocking(LatchType.REBALANCE);
 
+      cache(0).getAdvancedCache().getAsyncInterceptorChain()
+            .addInterceptorAfter(new AssertNoRetryInterceptor(), StateTransferInterceptor.class);
+
       //remote get is processed in current topology id.
       Future<Object> remoteGetFuture = remoteGet(cache(0), key);
       rpcManager0.waitForCommandToBlock();
 
-      NewNode joiner = addNode();
+      NewNode joiner = addNode(null, cb -> cb.customInterceptors().addInterceptor()
+         .position(InterceptorConfiguration.Position.FIRST).interceptor(new FailReadsInterceptor()));
```

Comment:
I'm not sure this test should test what happens when a read fails on a node that should have the data based on its current topology, even if (I'm assuming) this lets you simplify the test somewhat.